CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contribution guidelines
+
+__Things to do__
+
+0) Adhere to the [Code of Conduct](https://github.com/Attnam/ivan/blob/master/CODE_OF_CONDUCT.md).
+1) Join the [Attnam.com](https://attnam.com) forums, say hello. Figure out the changes the community wants to see, [check here as a starting point](https://attnam.com/wiki/Development_plans).
+2) Fix bugs before coding more bugs.
+3) Code new stuff while avoiding to code bugs.
+4) Release early and release often.
+5) Make experimental contributions while revising or deleting stuff that isn't working.
+
+__Awesome jobs__
+
+ - Fixing bugs and improving stability.
+ - Coding new subsystems, for example bodypart mutation subsystem, new dangers for player characters.
+ - Improving continuous integration and both cloud- and ground-based compiling systems for all supported platforms.
+ - Scripting new dungeons and new places for players to go.
+ - Commenting the code, document the danger system, weapon balance and combat system.
+
+__Forbidden jobs__
+
+ - Intentionally creating bugs.
+ - Upsetting game balance.
+ - Adding new materials.
```

Comment:
Well, this should have a footnote: Except when necessary or aesthetically pleasing (like new colours of ground for Urok :P ).